Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Linq;
+using MediaBrowser.Common.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Server.Migrations
+{
+    /// <summary>
+    /// The class that knows how migrate between different Jellyfin versions.
+    /// </summary>
+    public sealed class MigrationRunner
+    {
+        /// <summary>
+        /// The list of known migrations, in order of applicability.
+        /// </summary>
+        internal static readonly IUpdater[] Migrations =
+        {
+            new Routines.DisableTranscodingThrottling()
+        };
+
+        /// <summary>
+        /// Run all needed migrations.
+        /// </summary>
+        /// <param name="host">CoreAppHost that hosts current version.</param>
+        /// <param name="loggerFactory">Factory for making the logger.</param>
+        public static void Run(CoreAppHost host, ILoggerFactory loggerFactory)
+        {
+            var logger = loggerFactory.CreateLogger<MigrationRunner>();
+            var migrationOptions = ((IConfigurationManager)host.ServerConfigurationManager).GetConfiguration<MigrationOptions>(MigrationsListStore.StoreKey);
+
+            if (!host.ServerConfigurationManager.Configuration.IsStartupWizardCompleted)
```

Comment:
Okay, one last suggestion: What if here we do 
```
IsStartupWizardCompleted && **migrationOptions.Applied.Length == 0**
```
I think adding the extra check could future proof this to changes to the startup wizard behaviour.

Reply:
makes sense